homeassistant/components/tedee/__init__.py

```diff
@@ -38,6 +53,46 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = coordinator
 
+    async def unregister_webhook(_: Any) -> None:
+        await coordinator.async_unregister_webhook()
+        webhook_unregister(hass, entry.data[CONF_WEBHOOK_ID])
+
+    async def register_webhook() -> None:
+        # first make sure we don't have leftover callbacks to the same instance
+        instance_url = get_url(hass, allow_ip=True, allow_external=False)
+        try:
```

Comment:
The comment is misleadingly placed IMHO
```suggestion
        instance_url = get_url(hass, allow_ip=True, allow_external=False)
        # first make sure we don't have leftover callbacks to the same instance
        try:
```